spec/services/report/export_spec.rb

```diff
@@ -135,6 +136,15 @@
       ]
     end
 
+    let(:actuals) do
+      [
+        Transaction.new(financial_quarter: 1, financial_year: 2020, value: 20, parent_activity: activity),
+        Transaction.new(financial_quarter: 2, financial_year: 2020, value: 40, parent_activity: activity),
+        Transaction.new(financial_quarter: 3, financial_year: 2020, value: 80, parent_activity: activity),
```

Comment:
Would it be clearer if you instantiated 3 `Actual`s objects here rather than using the `Transaction` class?